Description: Do not build bundled libraries.
Author: Orion Poplawski <orion@nwra.com>
Abstract:
 This commit has been submitted by Orion in two
 portions. One was submitted to X2Go BTS and created
 on Wed, 10 Jul 2013.
 .
 The other portion has been taken from the Fedora
 package by Mike Gabriel and worked into this
 patch on Fri, 06 Dec 2013.
---
 nx-X11/config/cf/host.def         | 10 +++++++---
 nx-X11/lib/font/include/Imakefile |  2 +-
 2 files changed, 8 insertions(+), 4 deletions(-)

--- a/nx-X11/config/cf/host.def
+++ b/nx-X11/config/cf/host.def
@@ -311,6 +311,7 @@
  *
 #define BuildFreeType		NO
  */
+#define BuildFreeType		NO
 
 /*
  * To set non-default build options for the underlying FreeType libraries,
@@ -892,7 +893,7 @@
  *
 #define HasFreetype2 NO
  */
-#define HasFreetype2 NO
+#define HasFreetype2 YES
 
 /* 
  * To use a FreeType library already installed outside the default search
@@ -909,7 +910,7 @@
  *
 #define HasFontconfig NO
  */
-#define HasFontconfig NO
+#define HasFontconfig YES
 
 /* 
  * To use a Fontconfig library already installed outside the default search
@@ -925,7 +926,7 @@
  *
 #define HasExpat NO
  */
-#define HasExpat NO
+#define HasExpat YES
 
 /* 
  * To use a Expat library already installed outside the default search
@@ -961,7 +962,7 @@
 
 #define SharedLibXdmcp         YES
 
-#define BuildXaw7              YES
+#define BuildXaw7              NO
 
 #else /* #if !defined(NXZaurusXServer) && !defined(NXiPAQXServer) && !defined(DarwinArchitecture) */
 
@@ -1019,3 +1020,11 @@
 #endif
 
 #endif /* #if !defined(NXZaurusXServer) && !defined(NXiPAQXServer) && !defined(DarwinArchitecture) */
+
+#define BuildRman NO
+#define BuildFontEncLib NO
+#define BuildXcursorLibrary NO
+#define BuildXftLibrary NO
+#define BuildXft1Library NO
+#define SharedLibFont NO
+#define BuildLoadableXlibI18n NO
--- a/nx-X11/lib/font/include/Imakefile
+++ b/nx-X11/lib/font/include/Imakefile
@@ -1,5 +1,5 @@
 XCOMM $Xorg$
-    HEADERS = bitmap.h bufio.h fntfil.h fntfilio.h fntfilst.h fontenc.h \
+    HEADERS = bitmap.h bufio.h fntfil.h fntfilio.h fntfilst.h \
 	      fontencc.h fontmisc.h fontmod.h fontshow.h fontutil.h \
 	      fontxlfd.h
 all::
--- a/nx-X11/lib/Imakefile
+++ b/nx-X11/lib/Imakefile
@@ -213,14 +213,8 @@
 
 #if BuildLibraries
 OLDXLIBDIR = oldX
-ICELIBDIR = ICE
-SMLIBDIR = SM
-XTLIBDIR = Xt
-XMULIBDIR = Xmu
-XMUULIBDIR = Xmuu
 XPMLIBDIR = Xpm
 XTSTLIBDIR = Xtst
-FSLIBDIR = FS
 #endif
 
 
